packages/drivers/odsp-driver/src/odspPublicUtils.ts

```diff
@@ -16,6 +16,7 @@ export async function getHashedDocumentId(driveId: string, itemId: string): Prom
 
 /**
  * @alpha
+ * @deprecated - This is deprecated.
  */
 export interface ISnapshotContents {
 	snapshotTree: ISnapshotTree;
```

Comment:
Am I correct in assuming that all usage from shipping code is removed, but we kept interface in case someone else uses it? If so, should we just remove it (it's marked "alpha" :))

Reply:
Started using new interface in driver